CONTRIBUTING.md

```diff
@@ -636,3 +732,63 @@ Each branch is [deployed](https://github.com/CrowdStrike/glide-core/blob/main/.g
 And artifacts from our latest publish are [deployed](https://github.com/CrowdStrike/glide-core/blob/main/.github/workflows/on-merge-branch-changeset-release-main.yml#L95) to the top level of the bucket.
 It's a simple setup.
 But it means we can't have branch names that conflict and thus overwrite top-level directories that contain artifacts.
+
+### How does ARIA snapshot testing fit into overall accessibility testing?
+
+We use Axe to test accessibility.
+But Axe only tells us that a component's accessibility tree doesn't contain any obvious violations and that the accessibility tree is valid.
+What Axe doesn't tell us is whether the accessibility tree is correct.
+
+Dropdown, for example, when open, should show up as expanded in the accessibility tree.
+But, if it doesn't, Axe won't throw because Dropdown's accessibility tree is valid when Dropdown is open even if it's not expanded in the tree.
+
+Tracking a component's accessibility tree from commit to commit helps with issues like that and more.
+And it gives us confidence when making changes to a component.
+If a component's accessibility tree remains unchanged after the component is changed, we can more or less be sure that the component is just as accessible as it was prior to the change.
+
+That said, ARIA tests do not obviate manual testing with screenreaders.
+Because there's no guarantee that what's in the accessibility tree is interpreted correctly by screenreaders or that a screenreader hasn't had a regression.
+But it gets us about halfway there, and it greatly reduces continual and time-consuming VoiceOver passes that we'd otherwise have to do.
```

Comment:
```suggestion
But it gets us about halfway there, and it greatly reduces continual and time-consuming screenreader passes that we'd otherwise have to do.
```

Reply:
Don't forget the `.` to the end of your suggestion!